src/components/panel/panel.spec.js

```diff
@@ -2823,12 +2837,12 @@ describe('$mdPanel', function() {
     attachedElements.push(element);
   }
 
-  function clickPanelContainer() {
+  function clickPanelContainer(container) {
     if (!panelRef) {
       return;
     }
 
-    var container = panelRef.panelContainer;
+    var container = container || panelRef.panelContainer;
```

Comment:
@ThomasBurleson I think that there was some confusion as we extended this function. Previously, the container variable was declared internally and assigned to the `panelRef.panelContainer`. Now, you are able to pass in a container to click on. But, if you don't pass one in, it needs to still assign it to the default `panelRef.panelContainer`.